packages/libs/lambda-at-edge/src/default-handler.ts

```diff
@@ -272,7 +313,17 @@ const handleOriginRequest = async ({
     }
```

Comment:
I think this can be moved into the else block below just before the `await page.render(req, res)`?

Reply:
Does that mean we know for a fact that data requests will not result in a possible error page?

Reply:
Ah, I guess unmatched data request can still go to `pages/_error.js`, maybe still this at the start